ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -1975,9 +2001,12 @@ def test_nsschainvalidation_ipa_invalid_chain(self,
                 continue
             assert check["result"] == "ERROR"
             assert check["kw"]["dbdir"] == "%s/" % instance_dir
-            assert check["kw"]["msg"] == error_msg
             assert "marked as not trusted" in check["kw"]["reason"]
             assert check["kw"]["key"] == "%s:Server-Cert" % instance_dir
+            if parse_version(version) >= parse_version("0.6"):
+                assert check["kw"]["msg"] == error_msg
+            else:
+                assert error_msg_40_txt in check["kw"]["msg"]
```

Comment:
Similarly here. What 0.6 did was try to expand on the helpfulness of the error by adding additional context. reason in 0.6+ should == msg in 0.4.

Reply:
Modified the variable name to reason, found that only expected, got, msg are been returned in IPARAAgent check, hence reason check is not there.